detectPII.py

```diff
@@ -114,38 +114,68 @@ def detect_pii(self, text, language_code, filename):
         except ClientError as error:
             raise error
 
+def process_text(text, pdf_file, csv_writer):
+    comp_detect = ComprehendDetect()
+    if len(list(text)) == 0:
+        logger.info('No OCR text in {}.'.format(str(pdf_file)))
+    else:
+        for pdf_page in text:
+            if len(pdf_page.encode('utf-8')) > 5000:
+                page_parts = [pdf_page[:len(pdf_page)//2]] + [pdf_page[len(pdf_page)//2:]]
```

Comment:
See https://stackoverflow.com/questions/250357/truncate-a-string-without-ending-in-the-middle-of-a-word